addons/survey/security/survey_security.xml

```diff
@@ -35,24 +35,26 @@
             <field name="perm_read" eval="1"/>
             <field name="perm_create" eval="1"/>
         </record>
-        <record id="survey_survey_rule_survey_user_read" model="ir.rule">
-            <field name="name">Survey: officer: read all</field>
+
+        <record id="survey_survey_rule_survey_user_rw" model="ir.rule">
+            <field name="name">Survey: officer: read/write survey without responsible</field>
             <field name="model_id" ref="survey.model_survey_survey"/>
-            <field name="domain_force">[(1, '=', 1)]</field>
+            <field name="domain_force">[('user_ids', '=', False)]</field>
             <field name="groups" eval="[(4, ref('group_survey_user'))]"/>
             <field name="perm_unlink" eval="0"/>
-            <field name="perm_write" eval="0"/>
+            <field name="perm_write" eval="1"/>
```

Comment:
it's weird that you can write, but you can not unlink (make no sens imo)

I know it's like that so the user can add himself in the responsible, but so the unlink should be allowed